packages/e2e-test-utils/src/inserter.js

```diff
@@ -99,6 +99,12 @@ async function waitForInserterCloseAndContentFocus() {
 	);
 }
 
+async function waitForInserterSearch() {
+	// Inserter search results are debounced, let's wait a bit after typing.
+	// eslint-disable-next-line no-restricted-syntax
+	await page.waitForTimeout( 200 );
```

Comment:
Is there a better way achieve this? Like a wait for something from the results? It seems quite fragile and might even make the tests slower if we had a result sooner.

Reply:
Yes, thanks for pointing that out. 😉 I've been hacking to see if that solves it, but I'll definitely seek better ways to achieve it.

Reply:
Improved to wait for the results wrapper to appear in https://github.com/WordPress/gutenberg/pull/46153/commits/4ba3ee86e35514eaf78ec17d79e8a8bdec9b3d78.